packages/react-button/src/components/Button/ButtonBase.tsx

```diff
@@ -4,8 +4,8 @@ import { compose } from '@fluentui/react-compose';
 import { useButton } from './useButton';
 
 export const ButtonBase = compose<'button', ButtonProps, ButtonProps, {}, {}>(
-  (props, ref, composeOptions) => {
-    const { slots, slotProps } = useButton(props, composeOptions);
+  (props, ref, slotsAndState) => {
```

Comment:
i'm unsure of why this was renamed. I think it should still just be options.

Also the first value could be the computed state.

Reply:
These aren't the options anymore, but I guess I could pass this as part of the options object itself, I just saw no need to pass options if we're not calling state hooks on the render function, but let me know if I'm mistaken!